src/components/Header/Header.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useRef } from 'react';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  query: string;
+  todos: Todo[];
+  setQuery: (value: string) => void;
+};
+
+export const Header: React.FC<Props> = ({ query, todos, setQuery }) => {
+  const newTodoField = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    if (newTodoField.current) {
+      newTodoField.current.focus();
+    }
+  }, []);
+
+  return (
+    <header className="todoapp__header">
```

Comment:
we have header on top of the page!
we can have `head` of the app or `NewTodo` component (Checklist)